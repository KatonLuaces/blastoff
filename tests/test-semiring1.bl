def prints(M) {
    print(toString(M));
    return;
}

a = 2;
b = 3;
c = 0;

#arithmetic;
prints(a + b);
prints(a * b);
prints(a * c);

#logical;
prints(a + b);
prints(a * b);
prints(a * c);

#maxmin;
prints(a + b);
prints(a * b);
prints(a * c);
