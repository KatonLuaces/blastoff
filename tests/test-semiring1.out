5
6
0
1
1
0
3
2
0
